src/main/resources/config/liquibase/changelog/20230913163400_widget_aud.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet id="20230913163400-1" author="dbahazhkov">
+        <addColumn tableName="widget">
+            <column name="created_by" type="varchar(256)"/>
```

Comment:
The same question here, does this fields represented at the Domain objects? And where is this column filled?

Reply:
@pkev777 they are in the AbstractAuditinEntity abstract class